ros2bag/ros2bag/verb/play.py

```diff
@@ -93,6 +94,17 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '--start-offset', type=check_positive_float, default=0.0,
             help='Start the playback player this many seconds into the bag file.')
+        parser.add_argument(
+            '--wait-for-all-acked', type=check_not_negative_int, default=-1,
+            help='Wait until all published messages are acknowledged by all subscribers or until '
+                 'the timeout elapses in millisecond before play is terminated. '
+                 'Especially for the case of sending message with big size in a short time. '
```

Comment:
I feel like this flows a little better, though I tend to be a bit wordy.

```suggestion
                 'Especially useful when sending many messages with large sizes in a short timeframe.'
```